examples/with-imagekit/utils/imagekit.ts

```diff
@@ -0,0 +1,55 @@
+import ImageKit from "imagekit";
+import { FilterType } from "./types";
+import { FilterEnum } from "./enum";
+import { FileObject } from "imagekit/dist/libs/interfaces";
+
+const CONFIG_OPTIONS = {
+  publicKey: process.env.NEXT_PUBLIC_PUBLIC_KEY as string,
+  privateKey: process.env.PRIVATE_KEY as string,
+  urlEndpoint: process.env.NEXT_PUBLIC_URL_ENDPOINT as string,
+};
+
+export const listFiles = async (
+  limit = 10,
+  skip = 0,
+  filterState: FilterType,
+) => {
+  const imagekit = new ImageKit(CONFIG_OPTIONS);
+
+  const response = await imagekit.listFiles({
+    limit,
+    skip,
+    fileType: "all",
+    searchQuery:
+      filterState === FilterEnum.PHOTOS
+        ? '"private" = false AND (type="file") AND (format IN ["jpg","webp","png","gif","svg","avif","ico"])'
+        : filterState === FilterEnum.VIDEOS
+        ? '"private" = false AND (type="file") AND (format IN ["mp4","m4v","webm","mov"])'
+        : '"private" = false AND (type="file") AND (format IN ["mp4","m4v","webm","mov","jpg","webp","png","gif","svg","avif","ico"])',
```

Comment:
Don't use ternary conditions repeatedly. Becomes confusing to read. You can write a separate function with simple if-else or switch statement.

Reply:
Done, updated it.